### PR TITLE
fix: add missing shebang to `launchPackager.command`

### LINKED ISSUE #2095: launchPackager.command Exec format error
I'm getting the following error on my mac when running npm run android.

`Failed to execute process '..../react_native/node_modules/.bin/launchPackager.command'. Reason:
exec: Exec format error
The file '.../react_native/node_modules/.bin/launchPackager.command' is marked as an executable but could not be run by the operating system.
`

react-native-cli: 2.0.1
react-native: 0.72.5

As a workaround I add #!/bin/bash to the first line in launchPackager.command.
Any idea?

### PR DESCRIPTION
**Pull request by szymonrybczak**
Summary:
---------

Fixes #2095 

Test Plan:
----------
1. Clone the repository and do all the required steps from the [Contributing guide](https://github.com/react-native-community/cli/blob/main/CONTRIBUTING.md)
2. Run this command in Bash:

```sh
node /path/to/react-native-cli/packages/cli/build/bin.js run-ios
```

 

Checklist
----------

- [x] Documentation is up to date to reflect these changes.
- [x] Follows commit message convention described in [CONTRIBUTING.md](https://github.com/react-native-community/cli/blob/main/CONTRIBUTING.md#commit-message-convention)

### CODE REVIEW

**Comment by thymikee** [comment]
Looks like we hit test flakiness addressed here: https://github.com/react-native-community/cli/pull/2096 let's wait to merge it  first

File: packages/cli-tools/src/launchPackager.command [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/bin/sh
```
Comment (liamjones) [inline packages/cli-tools/src/launchPackager.command:1]: I don't believe `BASH_SOURCE` is going to be set in `sh`, this will probably need to be `bash`

Comment (szymonrybczak) [reply packages/cli-tools/src/launchPackager.command:1]: Ah, you're right 👍 Fixed.

**Comment by wilav-dev** [comment]
Hello. I'm installing from npm and still getting the old file version

**Pull request merged**